src/go/rpk/pkg/system/cpu.go

```diff
@@ -22,7 +22,7 @@ type CPUInfo struct {
 	Cores     int
 }
 
-func CpuInfo(fs afero.Fs) ([]*CPUInfo, error) {
+func GetCPUInfo(fs afero.Fs) ([]*CPUInfo, error) {
```

Comment:
Why `Get`?

Reply:
Because CPUInfo is already taken by the CPUInfo type.